src/Model/Contact.php

```diff
@@ -1587,7 +1587,7 @@ public static function getPostsFromUrl($contact_url, $thread_mode = false, $upda
 			return '';
 		}
 
-		if (in_array($contact["network"], [Protocol::ACTIVITYPUB, Protocol::DFRN, Protocol::DIASPORA, Protocol::OSTATUS, ""])) {
+		if (in_array($contact["network"], array_merge(Protocol::FEDERATED ,['']))) {
```

Comment:
Why the empty element?

Reply:
I'm currently unsure. But I wanted to make this PR as non breaking as possible, so I kept it.

Reply:
```suggestion
		if (empty($contact["network"]) || in_array($contact["network"], Protocol::FEDERATED))) {
```